site/app/templates/submission/EditTeamName.twig

```diff
@@ -4,16 +4,21 @@
 {% block body %}
     <p class="std-margin">
         Please fill in an appropriate name for your team if you would like.
+        <br>
+        Team names are limited to {{ team_name_length }} characters
```

Comment:
Most people aren't going to create team names which are excessively long.  As a result, I think adding help text like this to proactively prevent people from entering excessively long names is unnecessary.  Most people will understand the behavior of their browser when the max length attribute is enforced.

Reply:
Yes, agree with you
will just remove the line.